fetch-number-of-jobs-from-all-connected-controllers.groovy

```diff
@@ -0,0 +1,27 @@
+import com.cloudbees.opscenter.server.clusterops.steps.MasterGroovyClusterOpStep
+
+def retour = ''
+def stream = new ByteArrayOutputStream();
+def listener = new StreamBuildListener(stream);
+
+Jenkins.instance.getAllItems(com.cloudbees.opscenter.server.model.ConnectedMaster).each{
+    it.channel?.call(new MasterGroovyClusterOpStep.Script("""
+        class composedItem {
+            Object controllerName
```

Comment:
Why is controllerName an object?

Reply:
That's true. I have modified it to be a String, thanks!